packages/ui/src/components/Command/CommandMenuProvider.tsx

```diff
@@ -104,6 +104,7 @@ function useKeyboardEvents({
           return
         case 'k':
         case '/':
+          event.preventDefault()
```

Comment:
@dsukruth did this sneak in?

Reply:
Yes, It was a fix of #13339 I probably pushed both commits in the same PR.
**preventDefault()** should be called on the event object passed to the event listener function. This tells the browser to cancel the default action for that particular in this case which opens up the chrome search tab if not prevented.

Reply:
If you want me to revert it kindly let me know